src/execution/execute.js

```diff
@@ -605,12 +627,14 @@ function completeValueCatchingError(
   returnType: GraphQLType,
   fieldASTs: Array<Field>,
   info: GraphQLResolveInfo,
+  exePath: Array<string | number>,
   result: mixed
 ): mixed {
   // If the field type is non-nullable, then it is resolved without any
   // protection from errors.
   if (returnType instanceof GraphQLNonNull) {
-    return completeValue(exeContext, returnType, fieldASTs, info, result);
+    return completeValue(
+      exeContext, returnType, fieldASTs, info, exePath, result);
```

Comment:
small stylistic thing: typically when a function call doesn't fit in the 80c line, it has each argument presented on its own line for better legibility.